string      group
int8        angles_format
bool        random_start
---
string[]    joints
error		int8